bin/consusd

```diff
@@ -2,19 +2,76 @@
 
 'use strict';
 
-var daemon = require('daemonize2').setup({
-    main: '../index.js',
-    name: 'consusd',
-    pidfile: 'consusd.pid'
-});
-
-switch (process.argv[2]) {
-    case 'start':
-        daemon.start();
-        break;
-    case 'stop':
-        daemon.stop();
-        break;
-    default:
-        console.log('Usage: [start|stop]');
+// Capture an optional port from the command line args
+const PORT = process.argv.reduce((port, arg) => {
+    return (arg.match(/^--port=(\d+)$/) || [0, port])[1];
+}, 80);
+
+var path = require('path');
```

Comment:
Correct. This file is not being transpiled by babel.

I suppose we could make a special source folder with es6 and transpile it into `bin` with babel, but I don't know if that's worthwhile.